src/python/turicreate/toolkits/style_transfer/_tf_model_architecture.py

```diff
@@ -63,30 +76,30 @@ def define_instance_norm(tf_input, tf_index, weights, prefix):
     out: tensorflow.Tensor
         The instance norm output tensor to the network.
     """
-    epsilon=1e-5
-    
+    epsilon = 1e-5
+
     gamma = weights[prefix + "gamma_weight"]
     beta = weights[prefix + "beta_weight"]
-    
+
     tf_input_shape = tf_index.get_shape()
-    
+
     gamma_sliced = _tf.split(gamma, gamma.shape[0], axis=0)
     gamma_index = _tf.gather(gamma_sliced, tf_index)
     gamma_split = _tf.split(gamma_index, tf_input_shape[0], axis=0)
-    
+
     beta_sliced = _tf.split(beta, beta.shape[0], axis=0)
     beta_index = _tf.gather(beta_sliced, tf_index)
     beta_split = _tf.split(beta_index, tf_input_shape[0], axis=0)
-    
+
     input_split = _tf.split(tf_input, tf_input_shape[0], axis=0)
-    
+
     batch_norm_array = []
     for g, b, i in zip(gamma_split, beta_split, input_split):
-        inputs_shape = i.shape
+        # inputs_shape = i.shape
         inputs_rank = i.shape.ndims
-        
+
         reduction_axis = inputs_rank - 1
-        params_shape = inputs_shape[reduction_axis:reduction_axis + 1]
+        # params_shape = inputs_shape[reduction_axis:reduction_axis + 1]
```

Comment:
unused parameter, comment it.

Reply:
Why not just delete it?

Reply:
I want to, but I don't know if it's needed here.